reporter-web-app/src/components/SummaryView.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Copyright (C) 2017-2021 HERE Europe B.V.
```

Comment:
Commit message:

> so an user can quickly learn the amount

"a user", "learn about"

And you seem to have wrapped the commit message body text way before column 75.